pubspec.yaml

```diff
@@ -7,29 +7,30 @@ publish_to: 'none'
 version: 1.7.8+35
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: ">=2.17.0-0 <3.0.0"
```

Comment:
Why this change?

Reply:
I believe this excludes prerelease versions, but I'm not sure about this. This is taken from game_template repo, while trying to reduce differences between the repos, so that developers trying to understand the changes required over game_template, dont have to look at the differences/changes they don't need